stock_picking_auto_print/models/stock_picking.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2024 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import api, models
+
+from odoo.addons.web.controllers.main import clean_action
+
+_logger = logging.getLogger(__name__)
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def button_validate(self):
+        res = super().button_validate()
+
+        pickings_to_print = self.filtered(
```

Comment:
Why not implementing the original `_get_autoprint_report_actions` hook method?: https://github.com/OCA/OCB/blob/381c3681b55d185e2b27e8001edafe0dc284d6b4/addons/stock/models/stock_picking.py#L1658-L1665